test/test_board.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+from constans.constans import (
+    PLAYER_1,
+    PLAYER_2,
+    INITIAL_POSITION_PLAYER_1,
+    INITIAL_POSITION_PLAYER_2
+    )
+from game.board import Board
+from game.player import Player
+
+from constans.constants_game import (
+    LARGE,
+)
+
+
+class TestGame(unittest.TestCase):
+
+    def test_board_size(self):
+        board = Board()
+        self.assertEqual(len(board._board), LARGE)
+        self.assertEqual(len(board._board[0]), LARGE)
+
+    def test_place_character_initial_pos_player_1(self):
+        board = Board()
+        player_1 = Player(PLAYER_1)
+        board.place_character_initial_pos(
+            player_1.characters,
+            INITIAL_POSITION_PLAYER_1,
+            0)
+        for index, character in enumerate(player_1.characters):
+            row, col = INITIAL_POSITION_PLAYER_1[index]
+            self.assertEqual(board._board[row][col].character, character)
```

Comment:
Using `for` loops in tests is not a good practice. We can discuss this later.

Reply:
Ok, I think that in those cases I could test by a parameterized instead in a for loop